tests/Hilke.KineticConvolution.Tests/ConvolutionHelperTests.cs

```diff
@@ -11,58 +11,45 @@ namespace Hilke.KineticConvolution.Tests
     [TestFixture]
     public class ConvolutionHelperTests
     {
-        public static Point<DoubleNumber> MakePoint(double x, double y) =>
-            new Point<DoubleNumber>(DoubleNumber.FromDouble(x), DoubleNumber.FromDouble(y));
-
-        public static Tracing<DoubleNumber> MakeArc(
-            Fraction weight,
-            double centerX,
-            double centerY,
-            double directionStartX,
-            double directionStartY,
-            double directionEndX,
-            double directionEndY,
-            Orientation orientation,
-            double radius) =>
-            Tracing<DoubleNumber>.CreateArc(
-                1,
-                new Point<DoubleNumber>(DoubleNumber.FromDouble(centerX),
-                                        DoubleNumber.FromDouble(centerY)),
-                new DirectionRange<DoubleNumber>(
-                    new Direction<DoubleNumber>(
-                        DoubleNumber.FromDouble(directionStartX),
-                        DoubleNumber.FromDouble(directionStartY)),
-                    new Direction<DoubleNumber>(
-                        DoubleNumber.FromDouble(directionEndX),
-                        DoubleNumber.FromDouble(directionEndY)),
-                    orientation),
-                DoubleNumber.FromDouble(radius));
-
         [Test]
         public void When_Direction_Ranges_Are_Included_Then_Intersection_Should_Be_The_Innermost_Range()
         {
             const double radius1 = 2.0;
             const double radius2 = 2.0;
 
-            var arc1 = (Arc<DoubleNumber>)MakeArc(
-                1, 1.0, 2.0, 1.0, 0.0, 0.0, 1.0,
+            var convolutionFactory = new ConvolutionFactory();
+
+            var arc1 = convolutionFactory.CreateArc(
+                1,
+                1.0,
+                2.0,
+                1.0,
+                0.0,
+                0.0,
+                1.0,
```

Comment:
Ajouter les labels pour la lisibilité:

```suggestion
            var arc1 = convolutionFactory.CreateArc(
               weight:  1,
               centreX: 1.0,
               centreY: 2.0,
                ...
```

Reply:
Ca aussi, tu l'as pas fait sur ta branche?

Reply:
j'ai fait pour la factory, mais tu peux le laisser je m'en occupe dans l'autre branche.

Reply:
On fait comme ca, je resous pour l'instant.